pages/AddressBook.qml

```diff
@@ -114,6 +116,7 @@ Rectangle {
         anchors.leftMargin: 17
         anchors.rightMargin: 17
         anchors.topMargin: 5
+        placeholderText: qsTr("Saved to local wallet") + translationManager.emptyString
```

Comment:
All fields in address book are saved to local wallet. So maybe this placeholder isn't needed?